lib/pattern-importer.js

```diff
@@ -0,0 +1,155 @@
+var through = require('through2'),
+  utility = require('./utility'),
+  merge = require('lodash.merge'),
+  gutil = require('gulp-util'),
+  path = require('path'),
+  fs = require('fs'),
+  PluginError = gutil.PluginError,
+  PLUGIN_NAME = 'pattern-importer';
+
+/*
+ * Imports html patterns that are following the standards at github.com/pattern-library
+ *
+ * @param {Object} options
+*/
+exports.patternImporter = function (options) {
```

Comment:
@e2tha-e I haven't been able to figure that one out yet - as far as unit testing this kinda code goes. What I did was set up a new plibrary with the generator and change the gulpfile.js to point to this repo's ./lib/gulp-tasks/patterns-import.js gulp-task file. That means removing this line at the top:

```
  patternImporter = require('pattern-importer'),
```

and changing the way that the gulp tasks are set up from:

```
    patternImporter.gulpImportPatterns(gulp,npmImporterOptions);
```

to

```
    patternUtils.gulpImportPatterns(gulp,npmImporterOptions);
```